integration-cli/docker_cli_pull_test.go

```diff
@@ -369,3 +369,40 @@ func (s *DockerTrustSuite) TestTrustedPullWithExpiredSnapshot(c *check.C) {
 		}
 	})
 }
+
+// Test that pull continues after client has disconnected. #15589
+func (s *DockerTrustSuite) TestPullClientDisconnect(c *check.C) {
```

Comment:
Sorry to only notice this after the merge, but this should be probably be in DockerRegistrySuite.